js/image-auto-width.js

```diff
@@ -0,0 +1,19 @@
+(function($) {
```

Comment:
This looks good but is there a reason we don't use css rules for this?

Reply:
@jsmith176 this is a workaround of Site Executive. Every image has a specified width. We added a max-width: of 350px in the css to account for all sizes, however @danfox01 found that some images were much smaller than 350px and caused the images to be distorted on smaller screens. 

This script ensure that the width of the image does not go above the natural width of the image which in turn will ensure the image isn't distorted.

Reply:
Roger that